README.md

```diff
@@ -69,12 +68,31 @@ cd integreatly-operator
 
 If you are working against a fresh cluster it will need to be prepared using the following. 
 Ensure you are logged into a cluster by `oc whoami`.
-Include the `INSTALLATION_TYPE`. See [here](#variables-table) about this and other optional configuration variables.
+Include the `INSTALLATION_TYPE`. See [here](#3-configuration-optional) about this and other optional configuration variables.
 ```shell
 INSTALLATION_TYPE=<managed/managed-api> make cluster/prepare/local
 ```
 
-### 3. Run integreatly-operator
+
+### 3. Configuration (optional)
+
+If you are running RHOAM against a cluster which is smaller than the requirements listed above, you 
+should use the IN_PROW variable, otherwise the installation will not complete. 
+If you have a cluster which meets the requirements, this step can be skipped.
```

Comment:
Now, when it is not only about IN_PROW it looks like configuration with ll the variables only for clusters that do not meet requirements. The human might skip the whole step if a cluster is big. And we write about IN_PROW in two places now: here and in the table. We are saying here that the variable for a `cluster smaller than the requirements` and in table `for small clusters` - those are not equal (should have spotted it earlier, sorry). Not sure what is a proposition here. Probably, remove line 81 or write "if you are not happy with defaults". As for the IN_PROW, probably, move all details to the table, leaving here a warning (in bold?) with "installation might fail because of the cluster size - check IN_PROW in table"

Reply:
Yes, I take your point. I'm trying to be specific about the IN_PROW as it's the common one that catches folks out but we should also be generic as well. Also see my comment above. If you have any suggestion feel free to include it. I'm not sure what's best here tbh.